hls4ml/templates/vivado/nnet_utils/nnet_pooling.h

```diff
@@ -87,7 +87,8 @@ struct pooling1d_config{
   // IO size
   static const unsigned n_in = 10;
   static const unsigned pool_width = 2;
-  static const unsigned n_out = n_in / pool_width;
+  static const unsigned stride_width = 2;
+  static const unsigned n_out = (n_in - pool_width) / stride_width + 1;
```

Comment:
These values get overwritten so I would prefer if they were explicitly indicated as dummy instead of having a formula, but I think that is a minor point, and not really part of this PR.